ASoC: SOF: ipc3-control: Code cleanup for consistency and duplication

ujfalusi opened the pull request:
Hi,

rename the `snd_sof_refresh_control()` function to `sof_ipc3_refresh_control()` as it is ipc3 specific and only used locally and merge the bytes get_ext variant code (normal and volatile get) as the two function is 99% identical.

plbossart commented:
does anyone know if the failure in https://sof-ci.01.org/linuxpr/PR4131/build2679/devicetest/index.html?model=ADLP_RVP_NOCODEC_IPC4ZPH&testcase=multiple-pause-resume-5 is known?

It's unrelated to IPC3 but still that's rather spectacular.

ranj063 commented:
> does anyone know if the failure in https://sof-ci.01.org/linuxpr/PR4131/build2679/devicetest/index.html?model=ADLP_RVP_NOCODEC_IPC4ZPH&testcase=multiple-pause-resume-5 is known?
> 
> It's unrelated to IPC3 but still that's rather spectacular.

This is a known issue: https://github.com/thesofproject/sof/issues/6864

plbossart commented:
ok, thank you @ranj063, merging.

plbossart commented:
except that GitHub will not let me rebase with "This branch cannot be rebased due to too many changes"

jsarha reviewed (approved):
LGTM

ranj063 commented:
> except that GitHub will not let me rebase with "This branch cannot be rebased due to too many changes"

this makes no sense. THere are only 2 commits!

ujfalusi commented:
Heh, what is going on with GitHub nowdays? Let me force push this again...

ujfalusi commented:
Ah, the upstream merge went in?

ujfalusi commented:
Changes since v1:
- rebased on topic/sof-dev

@plbossart, @ranj063, @jsarha: this dropped your approval, can you check the PR again (the only change is the rebase)

jsarha reviewed (approved):
Appears to be a trivial rebase.

plbossart commented:
https://stackoverflow.com/questions/74523590/what-constitutes-a-branch-with-too-many-changes-for-github-pr-rebase-merging/74524897#74524897

So this is just the same issue we've been having since last week, delays in GitHub interaction impact results.

plbossart commented:
GitHub will still not rebase-and-merge this stupid branch, I see the 'Confirm rebase and merge' appear again after a 10s delay. Tried three times already.

plbossart commented:
5 times now. This is really broken.

dbaluta commented:
Ha, ha. Tried it too. It's broken for now.

kv2019i commented:
Tried as well, not working.

plbossart commented:
still not working, 3 hours later.

ujfalusi commented:
not working for me either :o

plbossart commented:
it's been a day and still not working for me either.

maybe if we start using expletives someone in GitHub support will notice? 

We've used GitHub for 4+ years and it's never been that bad. Someone missed their Q1 objectives already.

ujfalusi commented:
@plbossart, I have created a support ticket yesterday, I don't know how to make that visible for the other maintainers, let's see if that goes anywhere.

plbossart commented:
@ujfalusi I added a ticket as well two days ago, got a reply yesterday that they are looking into it. no news since. And it's still not working.

plbossart commented:
rebased-and-merged manually.

Wondering if GitHub maintains the FAA code?
